### ISSUE TITLE
Vector in the simulator

### ISSUE THREAD

**Issue by SenhorN** [mortbopet/Ripes#4]
The .word-declaration is defined as .word   expression [, expression]*  32-bit comma separated words.

The simulator Ripes don't support this?

because this code give-me a error:

```
> .data
> 
> **a: .word -7,8,107,11,13 #vector**
> 
> .text
```

**Comment by mortbopet** [comment]
Currently, there is no support for comma separated words words but as i read the RISCV asm documentation, comma separated words should be supported for .byte, .half, .word etc.
Thank you for finding the issue, ill have a look at it!

**Comment by mortbopet** [comment]
@SenhorN Fixed in 789f5b5a4d2c6f662bebeae984c38252d39a7a5d
